hedera-node/configuration/mainnet/application.properties

```diff
@@ -1 +1,12 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+staking.rewardRate=48630136986000
+tokens.nfts.maxAllowedMints=5_000_000
```

Comment:
@rbair23 do you want this value to remain here as an "override"? The same value is set in `src/main/resources/bootstrap.properties` below so it is not necessary.

Reply:
We should only have overrides in this file. The only setting in here that isn't a default is contracts.knownBlockHash. Do we even need to set it anymore?